pkgs/applications/networking/browsers/mullvad-browser/default.nix

```diff
@@ -104,6 +115,8 @@ let
       ];
       hash = "sha256-TAtBlSkfpqsROq3bV9kwDYIJQAXSVkwxQwj3wIYEI7k=";
     };
+    aarch64-darwin = darwinFetch;
+    x86_64-darwin = darwinFetch;
```

Comment:
Not sure if this works with `update-source-version` in the update script, but I'd hope it is.

Reply:
See https://github.com/NixOS/nixpkgs/pull/283874#discussion_r1468226947

Reply:
Once the update script is adjusted we can test if it works.